homeassistant/components/sensor/recorder.py

```diff
@@ -137,6 +156,47 @@ def _time_weighted_average(
     return accumulated / (end - start).total_seconds()
 
 
+def _time_weighted_circular_mean(
```

Comment:
Let's rename the function `_time_weighted_average` to `_time_weighted_arithmetic_mean`